src/test/cli/rbd/help.t

```diff
@@ -151,15 +151,14 @@
       watch                             Watch events on image.
   
   Optional arguments:
-    -c [ --conf ] arg     path to cluster configuration
-    --cluster arg         cluster name
-    --id arg              client id (without 'client.' prefix)
-    --user arg            client id (without 'client.' prefix)
-    -n [ --name ] arg     client name
-    -m [ --mon_host ] arg monitor host
-    --secret arg          path to secret key (deprecated)
-    -K [ --keyfile ] arg  path to secret key
-    -k [ --keyring ] arg  path to keyring
+    -c [ --conf ] arg                   path to cluster configuration
+    --cluster arg                       cluster name
+    --id arg                            client id (without 'client.' prefix)
+    --user arg                          client id (without 'client.' prefix)
```

Comment:
Nit: I think `--user` is essentially deprecated as well.

Reply:
done